ksql-functional-tests/src/main/java/io/confluent/ksql/test/model/InputRecordsNode.java

```diff
@@ -27,10 +27,11 @@
 
   public InputRecordsNode(
       @JsonProperty("inputs") final List<RecordNode> inputRecords
-  ) {
-    this.inputRecords = inputRecords == null
-        ? ImmutableList.of()
-        : ImmutableList.copyOf(inputRecords);
+  ) throws Exception {
+    if (inputRecords == null) {
+      throw new Exception("No 'inputs' field in the input file.");
```

Comment:
I'm not too familiar with our conventions around exceptions, but should this be `KsqlException` instead?

Reply:
Hmm, since it's related to the JSON parsing I just kept it as Exception.